build.gradle

```diff
@@ -34,6 +34,9 @@ dependencies {
     implementation 'io.jsonwebtoken:jjwt-api:0.11.5'
     implementation 'io.jsonwebtoken:jjwt-jackson:0.11.5'
 
+// https://mvnrepository.com/artifact/com.vladmihalcea/hibernate-types-52
+    implementation 'com.vladmihalcea:hibernate-types-52:2.19.2'
```

Comment:
what is this lib used for?

Reply:
This lib is used for supporting /models/Order.java JasonBinary type
`@TypeDef(name = "jsonb", typeClass = JsonBinaryType.class)`